.github/workflows/depends.yml

```diff
@@ -1,102 +0,0 @@
-name: Docker Depends Image
-
-on:
-  push:
-    branches:
-      - develop
-      - master
-  workflow_dispatch:
-
-  # this runs on the develop branch
-  schedule:
```

Comment:
correct, goal is to have those build on a schedule like before.

Reply:
What I wanted to do was use "latest" for the documentation so that it will use that in https://github.com/PecanProject/pecan-documentation

I did notice that the documentation at https://pecanproject.github.io/documentation does not match the documentation in that repository.